README.md

```diff
@@ -1,31 +1,36 @@
-# Web3 App
+# Lunaria
 
-Built on preact-es2020-template
+A decentralized asset manager
```

Comment:
shouldn't it be wallet?

Reply:
We ended up with "A decentralized asset management tool"